Give preference to target-optimised functions over glibc's ones,
which in turn ahave precedence over generic ones.

--- uClibc.orig/libc/Makefile	2005-07-20 08:10:44.000000000 +0200
+++ uclibc/libc/Makefile	2005-07-28 13:33:40.000000000 +0200
@@ -59,7 +59,7 @@
 	$(AR) dN 2 $(LIBNAME) $$objs && \
 	$(AR) dN 2 $(LIBNAME) $$objs
 	@for objfile in obj.signal \
-	                obj.string.generic obj.string.$(TARGET_ARCH) obj.string \
+	                obj.string obj.string.generic obj.string.$(TARGET_ARCH) \
 	                obj.sysdeps.common obj.sysdeps.$(TARGET_ARCH) ; do \
 		if [ -e $$objfile ] ; then \
 			echo $(AR) $(ARFLAGS) $(LIBNAME) $$objfile ; \
